config/config-mysql_apache_jvm_statsd.yaml

```diff
@@ -214,16 +243,26 @@ processors:
       
       - metric_name: apache_sent_kilobytes_total
         action: update
-        new_name: agent.googleapis.com/apache/traffic
+        new_name: apache/traffic
         operations:
           - action: toggle_scalar_data_type
 
+  filter/host:
```

Comment:
I'd move the host metrics filter & transform processors to the top to match the order you have in the pipelines at the bottom of this file